assembl/static/js/app/routeManager.js

```diff
@@ -386,9 +386,13 @@ var routeManager = Marionette.Object.extend({
     });
   },
 
-  voteWidgetFromV2: function(id, arg) {
+  voteWidgetFromV2: function(id, arg, queryString, showInModal) {
     var that = this;
-    var collectionManager = CollectionManager(); 
+    showInModal = (showInModal === true) ? true : false;
```

Comment:
Could just do:
```
showInModal ? showInModal : false
```

Reply:
or !!showInModal but I prefer explicit for readability